packages/flutter/lib/src/cupertino/text_selection_toolbar_button.dart

```diff
@@ -107,23 +110,64 @@ class CupertinoTextSelectionToolbarButton extends StatelessWidget {
         return localizations.pasteButtonLabel;
       case ContextMenuButtonType.selectAll:
         return localizations.selectAllButtonLabel;
+      case ContextMenuButtonType.liveTextInput:
       case ContextMenuButtonType.delete:
       case ContextMenuButtonType.custom:
         return '';
     }
   }
 
+  Icon? _getButtonIcon(BuildContext context) {
+    if (buttonItem == null) {
+      return null;
+    }
+    final IconData? iconData;
+    switch (buttonItem!.type) {
+      case ContextMenuButtonType.cut:
+      case ContextMenuButtonType.copy:
+      case ContextMenuButtonType.paste:
+      case ContextMenuButtonType.selectAll:
+      case ContextMenuButtonType.delete:
+      case ContextMenuButtonType.custom:
+        return null;
+      case ContextMenuButtonType.liveTextInput:
+        iconData = CupertinoIcons.doc_text_viewfinder;
+        break;
+    }
+
+    if (iconData == null) {
```

Comment:
Is this possible?  If not, you could get rid of this `if` block and make `iconData`'s type by non-nullable.